### PR TITLE
Give admins the ability to disable AppStore redirects

### PR DESCRIPTION
**Pull request by sethvargo**
Fixes https://github.com/google/exposure-notifications-verification-server/issues/1463

**Release Note**

```release-note
Give admins the ability to disable AppStore redirects for apps
```

/assign @whaught 

/cc @mikehelmick

### CODE REVIEW

File: pkg/database/migrations.go [hunk 1]
```diff
@@ -1900,3 +1911,13 @@ func (db *Database) MigrateTo(ctx context.Context, target string, rollback bool)
 	}
 	return m.Migrate()
 }
+
+// multiExec is a helper that executes the given sql clauses against the tx.
+func multiExec(tx *gorm.DB, sqls ...string) error {
```
Comment (sethvargo) [inline pkg/database/migrations.go:1916]: @whaught unrelated, but I got annoyed writing migrations...

File: cmd/server/assets/mobileapps/_app.html [hunk 2]
```diff
@@ -2,41 +2,75 @@
 
 {{$app := .app}}
 
+<style>
```
Comment (whaught) [inline cmd/server/assets/mobileapps/_app.html:5]: should this be in application.css?

File: pkg/database/migrations.go (hunk 1 above)
Comment (whaught) [reply pkg/database/migrations.go:1916]: nice - we do this a lot

**Review by whaught — APPROVED** [review]
/lgtm
/hold one question about css. feel free to unhold

File: cmd/server/assets/mobileapps/_app.html (hunk 2 above)
Comment (sethvargo) [reply cmd/server/assets/mobileapps/_app.html:5]: Meh, if we end up using it across multiple files we can move it over.

**Pull request merged**